docs/lance-rest-service.md

```diff
@@ -0,0 +1,394 @@
+---
+title: "Lance REST service"
+slug: /lance-rest-service
+keywords:
+  - Lance REST
+  - Lance datasets
+  - REST API
+license: "This software is licensed under the Apache License version 2."
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Overview
+
+The Lance REST service provides a RESTful interface for managing Lance datasets through HTTP endpoints. Introduced in Gravitino version 1.1.0, this service enables seamless interaction with Lance datasets for data operations and metadata management.
+
+The service implements the [Lance REST API specification](https://docs.lancedb.com/api-reference/introduction). For detailed specification documentation, see the [official Lance REST documentation](https://lance.org/format/namespace/rest/catalog-spec/).
+
+### What is Lance?
+
+[Lance](https://lance.org/format/) is a modern columnar data format designed for AI/ML workloads. It provides:
+
+- **High-performance vector search**: Native support for similarity search on high-dimensional embeddings
+- **Columnar storage**: Optimized for analytical queries and machine learning pipelines
+- **Fast random access**: Efficient row-level operations unlike traditional columnar formats
+- **Version control**: Built-in dataset versioning and time-travel capabilities
+- **Incremental updates**: Append and update data without full rewrites
+
+### Architecture
+
+The Lance REST service acts as a bridge between Lance datasets and applications:
+
+```
+┌─────────────────┐
+│   Applications   │
+│  (Python/Java)   │
+└────────┬────────┘
+         │ HTTP/REST
+         ▼
+┌─────────────────┐
+│  Lance REST      │◄──── Gravitino Metalake
```

Comment:
What does the direction of this arrow mean?

Reply:
This diagram seems inaccurate. LRS does not operate the dataset, but rather the Gravitino client.